packages/git/build.sh

```diff
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://git-scm.com/
 TERMUX_PKG_DESCRIPTION="Fast, scalable, distributed revision control system"
 TERMUX_PKG_LICENSE="GPL-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=2.35.0
-TERMUX_PKG_SRCURL=https://www.kernel.org/pub/software/scm/git/git-${TERMUX_PKG_VERSION}.tar.xz
-TERMUX_PKG_SHA256=47e677b475226857ceece569fb6ded8b85838ede97ae1e01bd365ac32a468fc8
+TERMUX_PKG_VERSION=2.35.1
+TERMUX_PKG_SRCURL=https://mirrors.edge.kernel.org/pub/software/scm/git/git-${TERMUX_PKG_VERSION}.tar.xz
```

Comment:
```suggestion
TERMUX_PKG_SRCURL=https://mirrors.kernel.org/pub/software/scm/git/git-${TERMUX_PKG_VERSION}.tar.xz
``` 

Any reason for adding this `.edge` subdomain to the URL. Both seem to hold the same data. And the checksums are also unaltered

Reply:
Solved!